l10n_ch_base_bank/i18n/de.po

```diff
@@ -6,10 +6,10 @@
 # OCA Transbot <transbot@odoo-community.org>, 2017
 msgid ""
 msgstr ""
-"Project-Id-Version: Odoo Server 10.0\n"
+"Project-Id-Version: Odoo Server 11.0\n"
```

Comment:
I'm not sure we should pull these changes... I'd expect them to be already in v11. @yvaucher ?